src/test/resources/edu/byu/cs240/checkstyle/readability/testInputs/commentedCode/should_FindNoErrors_when_commentInString.java

```diff
@@ -0,0 +1,50 @@
+import java.util.Scanner;
+
+class should_FindNoErrors_when_TextBlock {
+    public void notBananas() {
+        String notABananaToken = "/*";
+        String alsoNotABananaToken = "/**";
+        String stillNotABananaToken = "//";
+        String definitelyNotABananaToken = "///";
+        String dontEvenThinkAboutThisBeingABananaToken = "////";
+
+        String notABananaToken_twice = "/*" + "/*";
+        String alsoNotABananaToken_twice = "/*" + "/*";
+        String stillNotABananaToken_twice = "//" + "//";
+        String definitelyNotABananaToken_twice = "///" + "///";
+        String dontEvenThinkAboutThisBeingABananaToken_twice = "////";
+
+        String notABananaToken_withEndChar = "/*;";
+        String alsoNotABananaToken_withEndChar = "/**;";
+        String stillNotABananaToken_withEndChar = "//;";
+        String definitelyNotABananaToken_withEndChar = "///;";
+        String dontEvenThinkAboutThisBeingABananaToken_withEndChar = "////;";
+
+        String notABananaToken_twice_withEndChar = "/*;" + "/*;";
+        String alsoNotABananaToken_twice_withEndChar = "/**;" + "/**;";
+        String stillNotABananaToken_twice_withEndChar = "//;" + "//;";
+        String definitelyNotABananaToken_twice_withEndChar = "///;" + "///;";
+        String dontEvenThinkAboutThisBeingABananaToken_twice_withEndChar = "////;" + "////;";
```

Comment:
🎶 🕺 "You _ want _ my _ banana? You _ can't _ have _ my _ banana!" — Sung to the tune of ["PB&J" from EFY/FSY](https://youtu.be/3YedczKa1f4?si=KVYCdQr8BLZSuh0s&t=3) 🕺